mne/preprocessing/maxwell.py

```diff
@@ -449,6 +449,7 @@ def maxwell_filter(raw, origin='auto', int_order=8, ext_order=3,
         raw_sss._data[pos_picks, start:stop] = out_pos_data
 
     # Update info
+    info['dev_head_t'] = recon_trans  # set the reconstruction transform
```

Comment:
Potentially dump question: is this actually getting set in the `raw_sss` object or will it only change the `info` dict that we copied early on?